home/tests/test_content_import_export.py

```diff
@@ -327,6 +327,14 @@ def remove_buttons(page: DbDict) -> DbDict:
     return page
 
 
+@per_page
+def remove_example_values(page: DbDict) -> DbDict:
+    if "whatsapp_body" in page["fields"]:
+        for body in page["fields"]["whatsapp_body"]:
+            body["value"].pop("example_values", None)
+    return page
```

Comment:
We probably want to add the example values to the export and import, but happy for that to happen in another PR

Reply:
Cool, will do